test/common/schemaform/review/00-complete-form.e2e.spec.js

```diff
@@ -0,0 +1,97 @@
+const E2eHelpers = require('../../../e2e/e2e-helpers');
+const Timeouts = require('../../../e2e/timeouts.js');
+const HcaHelpers = require('../../../e2e/hca-helpers.js');
+const testData = require('../../../hca/schema/maximal-test.json');
+
+module.exports = E2eHelpers.createE2eTest(
+  (client) => {
+    HcaHelpers.initApplicationSubmitMock();
+
+    // Ensure introduction page renders.
+    client
+      .url(`${E2eHelpers.baseUrl}/health-care/apply/application`)
+      .waitForElementVisible('body', Timeouts.normal)
+      .assert.title('Apply for Health Care: Vets.gov')
+      .waitForElementVisible('.schemaform-title', Timeouts.slow)  // First render of React may be slow.
+      .click('.usa-button-primary');
+
+    E2eHelpers.overrideVetsGovApi(client);
+    E2eHelpers.overrideSmoothScrolling(client);
+    E2eHelpers.expectNavigateAwayFrom(client, '/introduction');
+
+    // Complete entire form
+    HcaHelpers.completeEntireForm(client, testData.data);
+
+    // Open first panel
+    client
+      .waitForElementVisible('.usa-accordion-bordered', Timeouts.normal)
+      .click('button.usa-button-unstyled')
+      .pause(1200);
+    client.expect.element('.edit-btn').to.be.visible;
+
+    // Edit form fields successfully
+    client
+      .click('.edit-btn');
+    client.expect.element('input[name="root_veteranFullName_first"]').to.be.visible;
+
+    client
+      .clearValue('input[name="root_veteranFullName_first"]')
+      .fill('input[name="root_veteranFullName_first"]', 'Jacques')
+      .click('.usa-button-primary');
+
+    client.expect.element('.review-row span').text.to.equal('Jacques');
+
+    // Edit form fields unsuccessfully
+    client.waitForElementVisible('.edit-btn', Timeouts.normal);
+    client
+      .click('.edit-btn');
+    client.expect.element('input[name="root_veteranFullName_first"]').to.be.visible;
+
+    client.clearElement('input[name="root_veteranFullName_first"]');
+    client.expect.element('input[name="root_veteranFullName_first"]').text.to.equal('');
+    client
+      .pause(1200)
+      .click('input[name="root_veteranFullName_middle"]');
+    client.expect.element('input[name="root_veteranFullName_first"]').text.to.equal('');
+
+    client
+      .click('.usa-button-primary')
```

Comment:
Once we click the "Update" button, the original value for the first name field (`Jacques`) reappears and the form successfully updates, when it should have failed.

Reply:
Strange, could another field such as service start date be tested instead?